lib/src/wip/autocomplete/popup.dart

```diff
@@ -50,13 +53,13 @@ class _PopupState extends State<Popup> {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: EdgeInsets.only(
-        left: min(widget.column, widget.editingWindowSize.width - width),
-        top: widget.row,
-      ),
-      child: ConstrainedBox(
+    return Positioned(
+      left: min(widget.column, widget.editingWindowSize.width - width),
+      top: widget.row,
```

Comment:
I moved this file to the `wip` dir because it is messy and needs refactoring, so please do not rely on anything here. One messy thing is that `row` and `column` here actually mean `y` and `x` in pixels although the names imply that these are characters.

Please fix such things as you notice them. If unsure, suggest such fixes first.